src/Thinktecture.Blazor.FileHandling/FileHandlingRelay.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.JSInterop;
+using Thinktecture.Blazor.FileHandling.Models;
+
+namespace Thinktecture.Blazor.FileHandling;
+
+public class FileHandlingRelay
+{
+    private readonly Lazy<ValueTask<IJSInProcessObjectReference>> _moduleTask;
+
+    public FileHandlingRelay(Lazy<ValueTask<IJSInProcessObjectReference>> moduleTask)
+    {
+        _moduleTask = moduleTask;
+    }
+    
+    [JSInvokable]
```

Comment:
Gibt es nicht eine Möglichkeit diesen Invoke Call von JS nach .NET zu vermeiden?
Aktuell reichst du ja nur die Action durch ohne mit der was im JS Code auszuführen.

Reply:
Ich sehe aktuell nicht, wie :/

Reply:
@patrickjahr Please check this again.

Reply:
Hab es mir nochmal angeschaut. Passt so für mich.